src/clustering.cr

```diff
@@ -172,9 +172,9 @@ class Clustering
 
   private def _stabilize(cluster_version : String, nodes : Array(HoundDog::Service::Node))
     Log.info { {node_event: "stablizing", version: cluster_version} }
-    stabilize.try &.call(nodes)
-    set_ready(cluster_version)
-    Log.info { {node_event: "stable", version: cluster_version} }
+    ready = stabilize.try &.call(nodes)
+    set_ready(cluster_version) if ready
+    Log.info { {node_event: ready ? "stable" : "failed to stabilize", version: cluster_version} }
```

Comment:
```suggestion
    node_event = if stabilize.try &.call(nodes)
      set_ready(cluster_version)
      "stable"
    else
      "failed to stabilize"
    end
    
    Log.info { {node_event: node_event, version: cluster_version} }
```

Reply:
that node event string won't be used if the log level is above info